src/prommis/uky/costing/tests/test_ree_costing.py

```diff
@@ -1898,11 +1954,14 @@ def test_HDD_Recycling_costing_noOM_usedefaults():
         fixed_OM=False,
     )
 
+    dt = DiagnosticsToolbox(m)
```

Comment:
This looks like it is one very large test - note that this can hide problems as the test will terminate at the first failure.